src/ngircd/conn-ssl.c

```diff
@@ -236,25 +236,40 @@ static bool
 Load_DH_params(void)
```

Comment:
Strictly speaking, I think there are more changes then really needed (like `goto out` instead if `return false` and `if (fp == NULL)`), and probably I would have moved `OSSL_DECODER_CTX_new_for_pkey(...)` below the `fopen(...)` to more closely follow the existing structure, but in the end I don't care much, all good, just my 5 cents, no changes needed, why not :smile: